src/site/content/en/blog/badging-api/index.md

```diff
@@ -159,41 +126,62 @@ then the "+" won't appear. No matter the actual number, just call
 `setAppBadge(unreadCount)` and let the user agent deal with
 displaying it accordingly.
 
-While the Badging API *in Chrome* requires an installed app, you shouldn't
+While the App Badging API *in Chrome* requires an installed app, you shouldn't
 make calls to the Badging API dependent on the install state. Just call the
 API when it exists, as other browsers may show the badge in other places.
 If it works, it works. If not, it simply doesn't.
 
-## Supporting both the origin trial and the stable API {: #supporting }
+## Setting and clearing the badge in the background from a service worker
 
-In the Chrome 79 and Chrome 80 [origin trial]({{origin_trial.url}}) the members
-of the badging API have different names than those in the current implementation. If
-you're participating in that origin trial, you'll need to support both the
-origin trail and the standard names. You can wrap the calls in the following
-functions. Once the origin trial is over or all your users have migrated to
-Chrome 81 or later, you can remove these functions from your code.
+In addition to setting the app badge while the page is open, it can also be
+set in the background by the service worker for the page, allowing it to be
+updated, even when the app isn't open. This can be done either through the
```

Comment:
```suggestion
You can set app badges form more than the open page. You can also set them in
the background using the service worker, allowing them to be update even when
the app isn't open. Do this either through the
```

Reply:
The problem I'm trying to correct is the shift in actor that happens between the subordinate and main clause. The subordinate clause that introduces the sentence implies an actor other than the app badge. In the main clause, the app badge is the actor. This becomes clear if you substitute the antecedent for the pronoun 'it'.

>In addition to setting the app badge, the app badge can also be set...

I'm not in love with my wording. If you have a better idea, I would only encourage you to stick with active voice.

Reply:
Thanks, rewrote this section.